src/python/tod/pointing.py

```diff
@@ -98,6 +136,11 @@ def __init__(self, pixels='pixels', weights='weights', nside=64, nest=False, mod
         # initialize the healpix pixels object
         self.hpix = hp.Pixels(self._nside)
 
+        if self._mode == "I":
+            self._nnz = 1
+        else:
+            self._nnz = 3
```

Comment:
We'll have to revisit this if we are mapping only the differenced time streams or we have spurious components.